inginious/frontend/plugins/analytics/pages/dashboard.html

```diff
@@ -0,0 +1,75 @@
+$def with() 
+
+$var title: Analytics
+<meta charset="utf-8">
+<style>
+    .calender-map {
+      font: 10px sans-serif;
+      shape-rendering: crispEdges;
+    }
+    .day {
+      stroke: #666;
+    }
+    .month {
+      fill: none;
+      stroke: #000;
+      stroke-width: 2px;
+    }
+    .RdYlGn .q0-11{fill:rgb(165,0,38)}
+    .RdYlGn .q1-{fill:rgb(215,48,39)}
+    .RdYlGn .q2-11{fill:rgb(244,109,67)}
+    .RdYlGn .q3-11{fill:rgb(253,174,97)}
+    .RdYlGn .q4-11{fill:rgb(254,224,139)}
+    .RdYlGn .q5-11{fill:rgb(255,255,191)}
+    .RdYlGn .q6-11{fill:rgb(217,239,139)}
+    .RdYlGn .q7-11{fill:rgb(166,217,106)}
+    .RdYlGn .q8-11{fill:rgb(102,189,99)}
+    .RdYlGn .q9-11{fill:rgb(26,152,80)}
+    .RdYlGn .q10-11{fill:rgb(0,104,55)}
+
+</style>
+
+<script src="https://d3js.org/d3.v3.min.js"></script>
+<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+
+<body>
+    <div style="display: flex">
+      <div style="margin: 5px">
+          <select id="duration_time" name="duration" onchange="location.reload()">
+              <option name="1" value=1 selected>Last year</option>
+              <option name="3" value=3>Last 3 year</option>
+              <option name="5" value=5>Last 5 year</option>
+          </select>
+
+      </div>
+
+      <div style="margin: 5px">
+        <select id="service" name="service" onchange="location.reload()">
+            <option name="all" value="" selected>All</option>
+            <option name="python_tutor" value="python_tutor">Python Tutor</option>
+            <option name="statistics" value="statistics">Statistics</option>
+            <option name="custom_input" value="custom_input">Custom Input</option>
+            <option name="linter" value="linter">Linter</option>
+        </select>
+
+    </div>
+
+      <div style="margin: 5px">
+        <text>Username</text><input id="student_username" name="username" type="text">
+        <button onclick="location.reload()">Update</button>
+      </div>
+
+  </div>
+
+    <div class="calender-map"></div>
+    <div id="myDiv"></div>
+    <div style="display: flex">
+      <div id="myDiv2" style="width: 50%"></div>
+      <div id="myDiv3" style="width: 50%"></div>
+    </div>
+    
+    <script src="static/calendar_view.js"></script>
```

Comment:
These scripts must be loaded in the python code using the already working hooks.